pylint/checkers/base/basic_checker.py

```diff
@@ -453,11 +453,14 @@ def visit_expr(self, node: nodes.Expr) -> None:
 
         # Warn W0133 for exceptions that are used as statements
         if isinstance(expr, nodes.Call):
-            inferred = utils.safe_infer(expr)
-            if isinstance(inferred, objects.ExceptionInstance):
-                self.add_message(
-                    "pointless-exception-statement", node=node, confidence=INFERENCE
-                )
+            name, _, _ = expr.as_string().partition("(")
+            exc_suffixes = ("Error", "Exc", "Exception", "Warning")
+            if any(name.endswith(suffix) for suffix in exc_suffixes):
```

Comment:
I would like a regex filter here. It's common that these are suffixes but not a guarantee. E.g. `ExceptionGroup`

Something like `.*(Error|Exc|Exception|Warning).*`.

The regex should also be precompiled at module level to improve performance.

--
_If added, user defined regexes could be checked here too then._

Reply:
Do you have a use case in mind that requires regexes?  Most of this could be achieved by changing from a `name.endswith(suffix)` pattern to `substring in name`.

Reply:
> Most of this could be achieved by changing from a `name.endswith(suffix)` pattern to `substring in name`.

Good point! That will work too. Just thought about combining it with the potential user input.